packages/ButtonGroup/stories/examples/Showcase.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { action } from "@storybook/addon-actions";
+import { boolean } from "@storybook/addon-knobs";
+import { Story, Rule, Tagline } from "storybook/assets/styles/common.styles";
+import { ShirtSizes } from "@paprika/helpers/lib/customPropTypes";
+import Heading from "@paprika/heading";
+import ButtonGroup from "../../src";
+
+function clickHandler() {
+  action("Clicked a button")();
+}
+
+const buttonGroupProps = () => ({
+  onChange: clickHandler,
+  hasIcons: boolean("hasIcons", false),
+  isDisabled: boolean("isDisabled", false),
+  isFullWidth: boolean("isFullWidth", false),
+  isSemantic: boolean("isSemantic", true),
+});
+
+const logSelected = selected => {
+  console.log(selected);
```

Comment:
Instead of logging to the console, in the Storybook it's nicer to use the `action()` addon.  You can still pass the result, `selected`, to be output.